ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -7,6 +7,157 @@
+      },
+      infoText: function(data, matches) {
+        if (data.me == matches.target) {
+          return {
+            en: 'Charge on YOU',
+          };
+        }
+        if (data.role != 'tank' || data.phase == 'both')
+          return;
+        return {
+          en: 'Tank Swap',
+        };
+      },
+    },
+    {
+      id: 'E6S Hands of Flame Cast',
+      regex: Regexes.ability({ source: ['Ifrit', 'Raktapaksa'], id: '4D00', capture: false }),
+      suppressSeconds: 1,
+      preRun: function(data) {
+        data.handsOfFlame = false;
+      },
+    },
+    {
+      id: 'E6S Instant Incineration',
+      regex: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4C0E' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E6S Meteor Strike',
+      regex: Regexes.startsUsing({ source: ['Ifrit', 'Raktapaksa'], id: '4C0F', capture: false }),
+      response: Responses.awayFromFront(),
+    },
+    {
+      id: 'E6S Hands of Hell',
+      regex: Regexes.headMarker({ id: '0016' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'Tether Marker on YOU',
+      },
+    },
+    {
+      id: 'E6S Hated of the Vortex',
+      regex: Regexes.startsUsing({ source: 'Garuda', id: '4F9F', capture: false }),
+      run: function(data) {
+        data.phase = 'both';
+      },
+    },
+    {
+      id: 'E6S Hated of the Vortex Effect',
+      regex: Regexes.gainsEffect({ effect: 'Hated of the Vortex' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Attack Garuda',
+      },
+    },
+    {
+      id: 'E6S Hated of the Embers Effect',
+      regex: Regexes.gainsEffect({ effect: 'Hated of the Embers' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Attack Ifrit',
+      },
+    },
+    {
+      id: 'E6S Raktapaksa Spawn',
+      regex: Regexes.ability({ source: 'Raktapaksa', id: '4D55', capture: false }),
+      run: function(data) {
+        data.phase = 'raktapaksa';
+      },
+    },
+    {
+      id: 'E6S Conflag Strike Knockback',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4C10', capture: false }),
+      response: Responses.knockback(),
+    },
+    {
+      id: 'E6S Irons Of Purgatory',
+      regex: Regexes.tether({ id: '006C' }),
+      condition: function(data, matches) {
+        return data.me == matches.target || data.me == matches.source;
+      },
+      alertText: function(data, matches) {
+        if (data.me == matches.source) {
+          return {
+            en: 'Tethered to ' + matches.target,
+          };
+        }
+        return {
+          en: 'Tethered to ' + matches.source,
+        };
+      },
+    },
+    {
+      id: 'E6S Conflag Strike Behind',
+      regex: Regexes.startsUsing({ source: 'Raktapaksa', id: '4C10', capture: false }),
+      delaySeconds: 31,
+      response: Responses.getBehind(),
```

Comment:
Maybe this was just what I've seen in practice, but I half wonder if this should be `getBehind('alarm')` just to be like really for real do this thing that's easy to forget.  Maybe that's silly.

Reply:
Eh, alert is probably ok for now.  I'll consider upping it later.